nuxeo-opencmis-impl/src/main/java/org/nuxeo/ecm/core/opencmis/impl/server/CMISQLtoNXQL.java

```diff
@@ -430,6 +435,19 @@ protected String getColumn(String propertyId) {
         if (propertyId.startsWith(CMIS_PREFIX) || propertyId.startsWith(NX_PREFIX)) {
             return getSystemColumn(propertyId);
         } else {
+            if (useElasticSearch && propertyId.indexOf(":") == -1) {
```

Comment:
The 2nd statement can be simplified to:

```java
!propertyId.contains(":")
```

Reply:
Please use the character-based `propertyId.indexOf(':')` instead, which is faster than the string version.